src/test/java/uk/gov/hmcts/sptribs/exception/InvalidResourceExceptionTest.java

```diff
@@ -10,15 +10,16 @@
 
 class InvalidResourceExceptionTest {
```

Comment:
@Serial annotation can be added to all the Exception classes for variable serialVersionUID